File: app/src/main/java/com/riders/thelab/data/local/model/RecyclerItem.java
package com.riders.thelab.data.local.model;

import com.riders.thelab.data.local.bean.RecyclerEnum;

import org.parceler.Parcel;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
@Parcel
public class RecyclerItem {

    String name;

    public RecyclerItem() {
    }

    public RecyclerItem(RecyclerEnum item) {
        this.name = item.getName();
    }
}

File: app/src/main/java/com/riders/thelab/data/local/model/RecyclerItem.kt
package com.riders.thelab.data.local.model

import android.os.Parcelable
import com.riders.thelab.data.local.bean.RecyclerEnum
import kotlinx.parcelize.Parcelize

@Parcelize
data class RecyclerItem constructor(var name: String? = null) : Parcelable {
    constructor(item: RecyclerEnum) : this() {
        name = item.fullName
    }
}